docs/change-log/2025-07-17-clickable-links-and-customizable-statuses-in-rich-presence.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Clickable Links and Customizable Statuses in Rich Presence"
+date: "2025-07-17"
+topics:
+- "Activities"
+- "Discord Social SDK"
```

Comment:
```suggestion
```

Sorry this one wasn't clear - can we remove the Social SDK category please, since it's not released on Social SDK yet,